### DISCUSSION TITLE
How to run a path and command with arguments only?

### DISCUSSION THREAD

**Discussion by karlvance** [luqasz/librouteros#137] [Q&A]
I've been attempting run to this string, that works when issued on my Mikrotik router: /interface/lte/monitor 0 once
Path is '/interface/lte', command is 'monitor' and it takes optional arguments, '0 once' in this case.  Without the options '0 once', it comes with a prompt, but I need to be able to parse it with code.

How does one code my string via this library?

**Comment by luqasz** [comment]
First you need to get `.id` and then:
```python
tuple(api('/interface/wireless/monitor', **{'.id': '*6', 'duration':1}))
```
